flink-formats/flink-avro-glue-schema-registry/pom.xml

```diff
@@ -33,7 +33,7 @@ under the License.
 	<packaging>jar</packaging>
 
 	<properties>
-		<glue.schema.registry.version>1.0.1</glue.schema.registry.version>
+		<glue.schema.registry.version>1.0.2</glue.schema.registry.version>
```

Comment:
Really this change (and below) should be in a dedicated PR. Can you please raise a separate PR with a dedicated Jira issue and we should be able to merge this right away.